schema/url/pass/plugin-pieplot.raml

```diff
@@ -0,0 +1,49 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  PieplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: PieplotSpec
+  
+  PieplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      showMissingness:
+        type: string
+        enum: ['TRUE', 'FALSE']
+        required: false
+      xAxisVariable: 
```

Comment:
i've been back and forth here. xAxis makes sense since that's where this var is mapped to in the bar plot. Overlay also makes sense because we're using color to stratify 'counts of participants' by this var. Any other reasons i'm missing that made xAxis more appropriate than overlay here?

Reply:
Idk, I guess in my head it's just a polar x-axis.. but we could just call it 'main' variable or something too of that's clearer generally

Reply:
Shah